crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4855,3 +4936,97 @@ impl From<AdyenStatus> for storage_enums::PayoutStatus {
         }
     }
 }
+
+impl<'a> TryFrom<(&domain::NetworkTokenData, Option<Secret<String>>)> for AdyenPaymentMethod<'a> {
+    type Error = Error;
+    fn try_from(
+        (token_data, card_holder_name): (&domain::NetworkTokenData, Option<Secret<String>>),
+    ) -> Result<Self, Self::Error> {
+        let adyen_network_token = AdyenNetworkTokenData {
+            payment_type: PaymentType::NetworkToken,
+            number: token_data.token_number.clone(),
+            expiry_month: token_data.token_exp_month.clone(),
+            expiry_year: token_data.get_expiry_year_4_digit(),
+            holder_name: card_holder_name,
+            brand: None,
+            network_payment_reference: None,
+        };
+        Ok(AdyenPaymentMethod::NetworkToken(Box::new(
+            adyen_network_token,
+        )))
+    }
+}
+
+impl<'a>
+    TryFrom<(
+        &AdyenRouterData<&types::PaymentsAuthorizeRouterData>,
+        &domain::NetworkTokenData,
+    )> for AdyenPaymentRequest<'a>
+{
+    type Error = Error;
+    fn try_from(
+        value: (
+            &AdyenRouterData<&types::PaymentsAuthorizeRouterData>,
+            &domain::NetworkTokenData,
+        ),
+    ) -> Result<Self, Self::Error> {
+        let (item, token_data) = value;
+        let amount = get_amount_data(item);
+        let auth_type = AdyenAuthType::try_from(&item.router_data.connector_auth_type)?;
+        let shopper_interaction = AdyenShopperInteraction::from(item.router_data);
+        let shopper_reference = build_shopper_reference(
+            &item.router_data.customer_id,
+            item.router_data.merchant_id.clone(),
+        );
+        let (recurring_processing_model, store_payment_method, _) =
+            get_recurring_processing_model(item.router_data)?;
+        let browser_info = get_browser_info(item.router_data)?;
+        let billing_address =
+            get_address_info(item.router_data.get_optional_billing()).transpose()?;
+        let country_code = get_country_code(item.router_data.get_optional_billing());
+        let additional_data = get_additional_data(item.router_data);
+        let return_url = item.router_data.request.get_return_url()?;
+        let card_holder_name = item.router_data.get_optional_billing_full_name();
+        let payment_method = AdyenPaymentMethod::try_from((token_data, card_holder_name))?;
+        let shopper_email = item.router_data.request.email.clone();
+        let shopper_name = get_shopper_name(item.router_data.get_optional_billing());
+        let mpi_data = MpiData {
+            directory_response: "Y".to_string(),
+            authentication_response: "Y".to_string(),
+            token_authentication_verification_value: token_data
+                .token_cryptogram
+                .clone()
+                .unwrap_or_default(),
+            eci: "07".to_string(),
```

Comment:
Are the `directory_response`, `authentication_response` and `eci` fields intentionally hardcoded?